src/aws/osml/image_processing/__init__.py

```diff
@@ -93,6 +122,17 @@
 """
 
 from .gdal_tile_factory import GDALTileFactory
+from .map_tileset import MapTile, MapTileId, MapTileSet
+from .map_tileset_factory import MapTileSetFactory, WellKnownMapTileSet
 from .sar_complex_imageop import histogram_stretch, quarter_power_image
 
-__all__ = ["GDALTileFactory", "histogram_stretch", "quarter_power_image"]
+__all__ = [
```

Comment:
Should we go back and standardize all our inits to use this __all__ pattern?

Reply:
The `__all__` list in an init.py defines the set of module names that are imported when someone imports *. The imagery toolkit modules are already making use of this as a way to explicitly define what is part of the public library API and what is not. 

In this PR if a consumer writes `from aws.osml.image_processing import *` then they will get the names listed here which deliberately excludes the WebMercatorQuadTileSet implementation of MapTileSet. The idea is that users should create these well known tile sets using the factory / enumerated IDs in WellKnownMapTileSet and never need to import or instantiate the concrete implementations directly.